core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.spec.ts

```diff
@@ -219,7 +228,7 @@ describe('TemplateBuilderComponent', () => {
 
             spectator.detectChanges();
 
-            store.init({
+            store.setState({
```

Comment:
Are we sure we need this whole tests here? Looks like is testing the store only.